magenta-lib/docs/magenta-lib/howto/fix-a-bad-deploy.md

```diff
@@ -0,0 +1,43 @@
+# Fixing a bad deploy
+
+Normally your Auto-Scaling Group will be  configured with a maximum capacity that is _twice_ the 'desired'
+capacity - this is to allow deploys to work by _doubling_ the desired capacity, bringing up new boxes, before
+killing the old ones off.
+
+When a Riff-Raff/Magenta deploy goes bad, your Auto-Scaling Group will often be left in a 'bad' state, where
```

Comment:
Happy to merge and then edit further, but I think it's worth explaining the rationale for why RiffRaff adopts this behaviour.